contrib/examples/generate-blueprint-docs-simple/GENERATED.md

```diff
@@ -0,0 +1,6 @@
+<!-- BEGINNING OF PRE-COMMIT-BLUEPRINT DOCS HOOK:TITLE -->
```

Comment:
What is the purpose of introducing these? To allow manual edition to live with the generated content?

IMO this file should be named something like `boilerplate.md`.

Reply:
Yes the goal was to allow generated content along side any other info a user may manually add.

It is not a `boilerplate` as the function inserts into the same md doc. Since this is an example, we have a `README.md` that instructs how to use the example. So I chose `GENERATED.md` to showcase this is the generated readme.

In real world use, the HTML tags will live in the README.md similar to `mdtogo` comments in the [kpt docs](https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/471a6a2dccd0f3e0da15fd55ccefe6a7e22a3fcb/functions/go/apply-setters/README.md?plain=1#L5) or [TF blueprints](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/dd9beb3da1bcc0ab2ac2a95c6ad7e238031ef33a/README.md?plain=1#L107).